src/Sulu/Bundle/ResourceBundle/Resources/public/js/components/condition-selection/auto-complete/main.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(['underscore'], function(_) {
+
+    'use strict';
+
+    var defaults = {
```

Comment:
i thinks it is better readable when the component is smaller. but i can move it there.